disttask/framework/framework_test.go

```diff
@@ -399,3 +540,42 @@ func TestSchedulerDownManyNodes(t *testing.T) {
 
 	distContext.Close()
 }
+
+func TestMultiTasks(t *testing.T) {
+	defer dispatcher.ClearDispatcherFactory()
+	defer scheduler.ClearSchedulers()
+	num := 3
+
+	m := make([]sync.Map, num)
+	RegisterTaskMeta(&(m[0]), &testDispatcherExt{})
+	RegisterTaskMeta1(&(m[1]), &testDispatcherExt{})
+	RegisterTaskMeta2(&(m[2]), &testDispatcherExt{})
+
+	distContext := testkit.NewDistExecutionContext(t, 1)
```

Comment:
```suggestion
	distContext := testkit.NewDistExecutionContext(t, 3)
```
More nodes will be better